.buildkite/release.sh

```diff
@@ -30,7 +30,7 @@ export YUKON_BACKEND_FULL_VERSION=$(grep __version__ src/yukon/backend/src/api/v
 export YUKON_FRONTEND_FULL_VERSION=$(node -e "console.log(require('./src/yukon/frontend/package.json').version);")
 export YUKON_BACKEND_MAJOR_MINOR_VERSION=$(echo $YUKON_BACKEND_FULL_VERSION | sed -E "s/([0-9]+\.[0-9]+)\.[0-9]+/\1/g")
 export YUKON_FRONTEND_MAJOR_MINOR_VERSION=$(echo $YUKON_FRONTEND_FULL_VERSION | sed -E "s/([0-9]+\.[0-9]+)\.[0-9]+/\1/g")
-tox
+tox -e py37-test,py38-test,py37-doctest,py38-doctest,jest,pylint,eslint,mypy,report,docs,py-package,npm-package
```

Comment:
You did this a bit backwards. You simply need to remove sonar-pr from the envlist in the tox.ini then it won't run by default. You can then add tox -e sonar-pr to the pr pipeline's yaml config.

Reply:
You are right (late work is not helping xD)